src/ReaderScreen/components/AudioPlayer/index.jsx

```diff
@@ -102,14 +101,15 @@ const AudioPlayer = ({ baniID, title, webViewRef }) => {
   };
 
   useEffect(() => {
-    if (isAudioAutoPlay && currentPlaying) {
+    if (currentPlaying) {
+      console.log("currentPlaying", currentPlaying);
```

Comment:
We can remove the debug console logs. There are few more in this diff, have a look